src/bioregistry/schema/struct.py

```diff
@@ -284,6 +284,10 @@ class Resource(BaseModel):
         description="The URI format string, which must have at least one ``$1`` in it",
         integration_status="required_for_new",
     )
+    rdf_uri_format: Optional[str] = Field(
+        title="RDF URI format string",
+        description="The RDF URI format string, which must have at least one ``$1`` in it",
```

Comment:
Consider linking in the description to the reference specification of the URI (or IRI)

Reply:
- URI format spec: https://www.rfc-editor.org/rfc/rfc3986
- IRI spec: https://www.ietf.org/rfc/rfc3987.txt

Reply:
Done in 0b2d923e